solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -852,6 +815,68 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         return outputAmount.add(_reserveAmount.mul(compN).div(compD));
     }
 
+    /**
+      * @dev returns the amount of tokens required for liquidation
+      *
+      * @param _poolToken       pool token
+      * @param _reserveToken    reserve token
+      * @param _poolAmount      pool token amount when the liquidity was added
+      * @param _targetAmount    target token amount received in return
+      * @param _isBaseAmount    true if the required amount is of the based token
+      * @return amount of tokens required for liquidation
+    */
+    function liquidationAmount(
+        IDSToken _poolToken,
+        IERC20Token _reserveToken,
+        uint256 _poolAmount,
+        uint256 _targetAmount,
+        bool _isBaseAmount)
+        internal view returns (uint256)
+    {
+        // calculate the amount of pool tokens required for liquidation
```

Comment:
this is the same as the function description so it's redundant. instead I'd just give context to the next line.